dle/Dockerfile.tests

```diff
@@ -5,63 +5,27 @@ WORKDIR /app
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
 
-RUN apt-get update --fix-missing
-# g++ for Pillow; unixodbc-dev for pyodbc
-# RUN apt-get install -y nginx python-psycopg2 postgresql graphviz gettext gdal-bin python-gdal python3-numpy gnupg gpg libpq-dev gcc g++ unixodbc-dev
-RUN apt-get install -y nginx python-psycopg2 postgresql libpq-dev gcc g++
-
-
-# Nginx tweaked
-# RUN addgroup --system nginx
-# RUN adduser --system nginx
-# RUN usermod -a -G nginx nginx
-# RUN mkdir -p /run/nginx
-
-# RUN mkdir -p /var/log/nginx && \
-#     ln -sf /dev/stdout /var/log/nginx/access.log && \
-#     ln -sf /dev/stdout /var/log/nginx/error.log
-
-# Use dev NGINX config
-# COPY nginx/nginx.dev.conf /etc/nginx/nginx.conf
-
-# install google chrome
-RUN apt-get install -y wget gnupg2
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
-RUN apt-get -y update
-RUN apt-get install -y google-chrome-stable
-
-# install chromedriver
-RUN apt-get install -yqq unzip curl
-RUN wget -O /tmp/chromedriver.zip http://chromedriver.storage.googleapis.com/`curl -sS chromedriver.storage.googleapis.com/LATEST_RELEASE`/chromedriver_linux64.zip
-RUN unzip /tmp/chromedriver.zip chromedriver -d /usr/local/bin/
-
-# set display port to avoid crash
+# g++ for Pillow
+RUN apt-get update --fix-missing \
+    && apt-get install -y nginx python-psycopg2 postgresql \
+    libpq-dev gcc g++ firefox-esr unzip curl
+
+# Install geckodriver for Firefox Selenium
+# Compatibility chart: https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html
+RUN curl -L https://github.com/mozilla/geckodriver/releases/download/v0.32.2/geckodriver-v0.32.2-linux64.tar.gz | tar xz -C /usr/local/bin
+RUN chmod +x /usr/bin/firefox
+# set display port to avoid Selenium crash
 ENV DISPLAY=:99
 
 # Python requirements
 COPY requirements.txt /app
-RUN pip install -r requirements.txt
+COPY requirements-dev.txt /app
+RUN pip install -r requirements-dev.txt
```

Comment:
Do we also install `requirements.txt` in order to run the tests in Docker?

Reply:
`requirements-dev` calls `requirements` in the first line: `-r requirements.txt`. So they do get installed.

Reply:
That's fairly typical, it does mean that the linting CI setup takes longer. We need everything for the Django tests, but only a couple packages for linting